contrib/network/connection_balance/dlb/source/connection_balancer_impl.cc

```diff
@@ -15,6 +15,27 @@ namespace Envoy {
 namespace Extensions {
 namespace Dlb {
 
+Envoy::Network::ConnectionBalancerSharedPtr DlbConnectionBalanceFactory::fallback(
+    const envoy::extensions::network::connection_balance::dlb::v3alpha::Dlb::FallbackPolicy
+        fallback_policy,
+    const std::string& message) {
+  switch (fallback_policy) {
+  case envoy::extensions::network::connection_balance::dlb::v3alpha::
+      Dlb_FallbackPolicy_ExactConnectionBalance: {
```

Comment:
You're using the mangled proto enum name, can you used the unmangled? Here and below.

Reply:
Updated.